### DISCUSSION TITLE
[Wiki] Installing yabai – fix sudoers user spec syntax

### DISCUSSION THREAD

**Discussion by alexwforsythe** [asmvik/yabai#1546] [General]
The sudoers user spec suggested in [Installing yabai (latest release) > Configure scripting addition](https://github.com/koekeishiya/yabai/wiki/Installing-yabai-(latest-release)#configure-scripting-addition) doesn't work for me on Ventura 13.0.1. It causes yabai to print the following error when started:
```
sudo: a terminal is required to read the password; either use the -S option to read from standard input or configure an askpass helper
sudo: a password is required
```

I was able to fix it by removing the spaces around the '=':
```diff
< <user> ALL = (root) NOPASSWD: sha256:<hash> <yabai> --load-sa
---
> <user> ALL=(root) NOPASSWD: sha256:<hash> <yabai> --load-sa
```

Let's update the wiki page to suggest the corrected spec instead!

P.S. – can we update the footer of the wiki to explain how we should propose article changes, or at least link to Discussions instead of #5?

**Comment by asmvik** [comment]
Fixed, thanks.
